src/config.ts

```diff
@@ -5,7 +5,9 @@ const throwConfigError = (key: string) => {
 // Note that including an API key in an app is not very secure, as the app can be
 // decompiled to access it. It's fine for this training exercise though.
 const apiKey = process.env.API_KEY ?? throwConfigError('API_KEY');
+const baseURL = 'https://mobile-api-softwire2.lner.co.uk';
```

Comment:
It's a good idea to add this to the config, but we might as well go the whole way and read it from an envionment variable like we do for the `apiKey` above. Even though it's not a secret, it's very useful to be able to do this so that we can configure our app to point at different versions of our backend in different environments without having to modify any code.